src/css/profile/mobile/common/popup.less

```diff
@@ -93,6 +88,144 @@
 	}
 }
 
+@media (min-width: 0px) and (max-width: 479px) {
+	@media (min-height: 600px) and (max-height: 959px) {
+		.ui-popup {
+			width: 100%;
+			margin: 0 auto;
+			background-color: rgba(255, 255, 255, 0);
+			margin-top: 40px;
+			margin-bottom: 16px;
+			padding-top: 16px;
+			.ui-popup-header {
+				font-size: 20px;
+				text-align: left;
+				padding: 24px 24px 18px 24px;
+				position: sticky;
+			}
+			.ui-popup-content {
+				font-size: 16px;
+				text-align: left;
+				padding: 0px 24px 24px 24px;
+				.ui-listview {
+					border-top-width: 1px;
+					border-top-color: #CCCCCC;
```

Comment:
I think it's better the color define in the theme.color.less and light.variable.less.
For example,  @dropdown-menu-options-border: #cccccc;

Reply:
I think you can refer to theme.color.less

//***************************************************************************
//						Popup
//***************************************************************************
@color_popup_bg: var(--popup-background); // #[color] background color
@color_popup_header_bg: var(--popup-background); // #[color] header background color
@color_popup_footer_bg: var(--popup-background); // #[color] footer background color
...

Reply:
This code should be modified to show the divider dynamically as you scroll later. 
It's not the definition you need right now.